src/modules/import/domain/import-event-handler.ts

```diff
@@ -104,14 +135,29 @@ export class ImportEventHandler {
                     `System hat einen neuen Benutzer ${savedPersonWithPersonenkontext.person.referrer} (${savedPersonWithPersonenkontext.person.id}) angelegt.`,
                 );
             } else {
-                return this.logger.error(
+                this.logger.error(
                     `System hat versucht einen neuen Benutzer für ${importDataItem.vorname} ${importDataItem.nachname} anzulegen. Fehler: ${savedPersonWithPersonenkontext.message}`,
                 );
+
+                importVorgang.fail();
+                // eslint-disable-next-line no-await-in-loop
+                await this.importVorgangRepository.save(importVorgang);
+
+                throw new ImportDomainError(
+                    `The creation of person with personenkontexte for the import transaction:${importVorgang.id} failed`,
+                    importVorgang.id,
+                );
             }
 
             if (!savedPersonWithPersonenkontext.person.newPassword) {
-                return this.logger.error(
-                    `Person with ID ${savedPersonWithPersonenkontext.person.id} has no start password!`,
+                this.logger.error(`Person with ID ${savedPersonWithPersonenkontext.person.id} has no start password!`);
+
+                importVorgang.fail();
+                // eslint-disable-next-line no-await-in-loop
+                await this.importVorgangRepository.save(importVorgang);
+                throw new ImportDomainError(
+                    `The creation for a password for the person with ID ${savedPersonWithPersonenkontext.person.id} for the import transaction:${importVorgang.id} has failed`,
+                    importVorgang.id,
```

Comment:
Just to understand, why do we throw an error if the person does not have a "newPassword"? and is it enough to just throw an unhandled exception without explicitly telling why it failed here?